inc/class-talk-settings-page.php

```diff
@@ -145,6 +154,40 @@ public function render_container_classes_field() {
 		<?php
 	}
 
+	/**
+	 * Prints drop down for version.
+	 *
+	 * @since 0.0.3
+	 */
+	public function render_version_field() {
+		?>
+		<select
+			style="width: 600px; height: 40px;"
+			name="coral_talk_version"
+			placeholder=""
+			id="coral_talk_version"
+			type="select"
+		>
+			<option value="4"
+				<?php 
+					if (esc_attr( get_option( 'coral_talk_version' ) ) === "4")
+						echo "selected=\"selected\""
+				?>
+			>
+				4
+			</option>
+			<option value="5"
+				<?php 
+					if (esc_attr( get_option( 'coral_talk_version' ) ) === "5")
+						echo "selected=\"selected\""
```

Comment:
I would suggest moving the `esc_attr( get_option( 'coral_talk_version' ) )` to the top of the `render_version_field` function if we can, then it should be good to merge!